### PR TITLE
Revert adoptisation checkouts

### PR DESCRIPTION
**Pull request by M-Davies**
* Partially reverts https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/commit/c47a59cfc3a7dbb3415d2cd59f53f2228680c9a6#

Signed-off-by: Morgan Davies <morgandavies2020@gmail.com>

### CODE REVIEW

**Comment by M-Davies** [comment]
run-tests

**Comment by M-Davies** [comment]
run-tests

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
Tester failed due to the bash scripts not existing in this repo. However, they are correctly checking out to ci now https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/job/build-test/job/jobs/job/jdk8u/job/jdk8u-linux-x64-hotspot/1038/console

File: .github/workflows/comment-bot.yml [hunk 1]
```diff
@@ -15,4 +15,4 @@ jobs:
           issue-number: ${{ github.event.pull_request.number }}
           body: |
             Thank you for creating a pull request!
-            In order to run the pipeline tests I require an admin to post the following comment: `run-tests`
+            In order to run the pipeline tests I require an admin to post the following comment: `run tests`
```
Comment (andrew-m-leonard) [inline .github/workflows/comment-bot.yml:18]: When I did my PR "run-tests" worked but "run tests" did not have we changed it now to be consistent across repos?

**Comment by andrew-m-leonard** [comment]
run-tests

**Comment by andrew-m-leonard** [comment]
run tests

File: .github/workflows/comment-bot.yml (hunk 1 above)
Comment (andrew-m-leonard) [reply .github/workflows/comment-bot.yml:18]: Just tried it and "run tests" I think did work

Comment (M-Davies) [reply .github/workflows/comment-bot.yml:18]: The regex on the job hasn't changed `.*(re)?run tests.*`. Maybe you had commented `run tests` earlier in the PR and it picked up that

**Comment by andrew-m-leonard** [comment]
Does PR tester need an import? Compile fails with RepoHandler not found

**Comment by M-Davies** [comment]
> Does PR tester need an import? Compile fails with RepoHandler not found

Ugh another reason why I want https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/issues/14 to happen. It's due to it not realising a stub is in place. I'll update the PR

**Comment by M-Davies** [comment]
Compile groovy is always going to fail in this case as we can't properly duplicate the RepoHandler.groovy class inside the ContextStubs

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 2]
```diff
@@ -1215,8 +1217,7 @@ class Build {
                             if (buildConfig.DOCKER_FILE) {
                                 try {
                                     context.timeout(time: buildTimeouts.DOCKER_CHECKOUT_TIMEOUT, unit: "HOURS") {
-                                        def repoHandler = new RepoHandler(context, USER_REMOTE_CONFIGS)
-                                        repoHandler.checkoutAdopt()
+                                        context.checkout context.scm
```
Comment (gdams) [inline pipelines/build/common/openjdk_build_pipeline.groovy:1220]: this will checkout ci-jenkins-pipelines rather than openjdk-build now?

Comment (andrew-m-leonard) [reply pipelines/build/common/openjdk_build_pipeline.groovy:1220]: I suspect this needs to now be:
if (useAdoptShellScripts) {
    repoHandler.checkoutAdopt()
} else {
    repoHandler.checkoutUser()
}

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

**Review by gdams — APPROVED** [review]

**Review by karianna — APPROVED** [review]
LGTM

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 3]
```diff
@@ -949,7 +949,11 @@ class Build {
 
             try {
                 context.timeout(time: buildTimeouts.NODE_CHECKOUT_TIMEOUT, unit: "HOURS") {
-                    repoHandler.checkoutAdopt()
+                    if (useAdoptShellScripts) {
+                        repoHandler.checkoutAdopt()
+                    } else {
+                        context.checkout context.scm
```
Comment (AdamBrousseau) [inline pipelines/build/common/openjdk_build_pipeline.groovy:955]: @M-Davies This seems to checkout 2 different repos depending on if the useAdoptShellScripts flag is enabled. checkoutAdopt will clone openjdk-build and checkout scm will clone ci-jenkins-pipeline. Unless I've misconfigured something on my side but I don't think this is what is expected.

Comment (M-Davies) [reply pipelines/build/common/openjdk_build_pipeline.groovy:955]: I think that's me getting confused on what exact repo we're checking out. I intend to address any issues as part of https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/issues/34

**Pull request merged**